### PR TITLE
Feat: Added Product listing page

### PR DESCRIPTION
**Pull request by LordUttam**
Click on carousel slide to visit products listing page. Thanks :)

### CODE REVIEW

File: pages/products.html [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Uttam Sharma">
+    <script src="https://use.fontawesome.com/6d5bbe7895.js"></script>
+    <link rel="stylesheet" href="https://kalpaui.netlify.app/styles/main.css">
+    <link rel="stylesheet" href="../styles/main.css">
+    <link rel="stylesheet" href="../styles/products.css">
+
```
Comment (Mohasin-Haque) [inline pages/products.html:13]: you can remove the whitespaces

**Review by Mohasin-Haque — APPROVED** [review]
Everything seems fine to me. You can go ahead.

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the Best.

File: pages/products.html (hunk 1 above)
Comment (LordUttam) [reply pages/products.html:13]: I prefer to have a line gap between links and title

**Pull request merged**